HackathonOpenStreetMap/OpenStreetMapLibrary/src/main/java/com/android/lib/map/osm/overlay/OsmPolygonOverlay.java

```diff
@@ -102,8 +100,9 @@ public void onZoomLevelChanges(View view) {
 		}
 	}
 	
-	private void offsetPath(Path path) {
-		Point startPoint = mStartPoints.get(path);
+	private void offsetPath(MapPolygon mapPolygon) {
+		Point startPoint = mStartPoints.get(mapPolygon);
+		Path path = mapPolygon.getPath();
```

Comment:
This looks like it may have been a bug, as the key of the map was MapPolygon, not path

Reply:
Good catch!